fds.go

```diff
@@ -104,7 +107,7 @@ func (f *Fds) Listen(network, addr string) (net.Listener, error) {
 		return ln, nil
 	}
 
-	ln, err = net.Listen(network, addr)
+	ln, err = f.lc.Listen(context.Background(), network, addr)
```

Comment:
the empty `net.ListenConfig` is used in `net.Listen` : https://golang.org/src/net/dial.go?s=21680:21734#L707

Reply:
That's ok.

You need to take care of Fds .ListenPacket though.